prboom2/src/dsda/zipfile.c

```diff
@@ -0,0 +1,135 @@
+//
+// Copyright(C) 2022 by Pierre Wendling
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// DESCRIPTION:
+//	DSDA zipfile support using libzip
+//
+
+#include "config.h"
+
+#include "../lprintf.h"
+
+#ifdef HAVE_LIBZIP
+#include "../m_file.h"
```

Comment:
Please organize headers and paths like so for consistency:
```c
#include <stdio.h>
#include <zip.h>

#include "m_file.h"
#include "z_zone.h"

#include "dsda/utility.h"
```

Reply:
Done in [9fb2b1f](https://github.com/kraflab/dsda-doom/pull/292/commits/9fb2b1f4ac196728626ebe9b0780f85d67da66bb).